spyder/plugins/updatemanager/workers.py

```diff
@@ -321,3 +328,20 @@ def start(self):
                 self.sig_ready.emit()
             except RuntimeError:
                 pass
+
+
+def _rate_limits(page):
```

Comment:
Please move this function to the beginning of the file, i.e. after `SSL_ERROR_MSG` and before `UpdateDownloadCancelledException`. It looks odd placed here.